src/constants.rs

```diff
@@ -40,6 +40,7 @@ pub fn test_announcements_enabled() -> bool {
 }
 
 pub const CASERVER_DIR: &str = "cas";
+pub const CA_OBJECTS_DIR: &str = "objects";
```

Comment:
Perhaps I'm not following correctly what happens but it seems to me that `CA_OBJECTS_DIR` is created under `data_dir`, and not under the `cas` dir, yet these objects apparently relate to a CA (given the name of the const). Is `"objects"` thus confusingly imprecise and might benefit from including the term "ca" or being a child of a CA specific parent directory?